lib/server.js

```diff
@@ -196,7 +205,7 @@ module.exports = function createServer(config, log) {
       fetchRecords(ip, email, phoneNumber)
         .spread(
           function (ipRecord, reputation, emailRecord, ipEmailRecord, smsRecord) {
-            if (ipRecord.isBlocked()) {
+            if (ipRecord.isBlocked() && ! isWhitelisted(ip, email)) {
```

Comment:
You'll have to pry that dangling bang from my cold, dead fingers! There's plenty of love for it round my neck of the woods!